$NetBSD: patch-Makefile,v 1.1 2021/10/24 07:47:41 nia Exp $

We need valac to honor CFLAGS and LDFLAGS so pkgsrc hardening
features are applied.

--- Makefile.orig	2021-10-20 00:56:11.000000000 +0000
+++ Makefile
@@ -13,7 +13,7 @@ LFLAGS   = `pkg-config --libs glib-2.0 g
 all: $(TARGET)
 
 $(TARGET): $(SRC)
-	$(VALAC) $(IFLAGS) $(SRC) -o $(TARGET)
+	$(VALAC) --cc="cc $(CFLAGS) $(LDFLAGS)" $(IFLAGS) $(SRC) -o $(TARGET)
 #	$(CC) $(CFLAGS) $(IFLAGS) $(SRC) $(LFLAGS) $(LDFLAGS) -o $(TARGET)
 
 install: $(TARGET)
